spec/requests/comments_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Comments', type: :request do
+  include Docs::V1::Comments::Api
+
+  let(:user) { create(:user) }
+  let(:project) { create(:project, user_id: user.id) }
+  let(:task) { create(:task, project_id: project.id) }
+  let(:headers) { authorization_header_for(user) }
+
+  describe 'POST #create' do
+    include Docs::V1::Comments::Create
+
+    context 'when success' do
+      let(:params) { { body: FFaker::Lorem.word } }
+      let(:request_comment) { post api_v1_task_comments_path(task, id: task), headers: headers, params: params }
```

Comment:
subject { post api_v1_task_comments_path(task, id: task), headers: headers, params: params }

expect { subject }.to ...